src/collective/nitf/upgrades/v1008/__init__.py

```diff
@@ -12,7 +12,7 @@ def fix_collections(context):
     logger.info('About to update {0} objects'.format(len(collections)))
     for col in collections:
         obj = col.getObject()
-        query = obj.getRawQuery()
+        query = obj.getQuery()
```

Comment:
@winstonf88 I think we can go just with getQuery method, since it will allways be available in collections: https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/content.py#L39-L40